### PR TITLE
pull request for peer feedback week1

### PR DESCRIPTION
**Pull request by abbyeckland**

### CODE REVIEW

**Comment by sebkopf** [comment]
Notebook review link: https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1

**Comment by sebkopf** [comment]
binder link for review: <a href='https://mybinder.org/v2/gh/CUB-Computational-Tools/2020-ps-aeckland2696/binder-python?urlpath=git-pull?repo=https%3A%2F%2Fgithub.com%2FCUB-Computational-Tools%2F2020-ps-aeckland2696%26branch%3Dweek1%26urlpath%3Dlab'></a>

File: week1.ipynb [hunk 1]
```diff
@@ -0,0 +1 @@
...uation to solve for $λ$.\n\n4) Your result should be pretty big. Convert your answer in meters back to miles. 1 m = 6.214E-4 miles.\n\n5) What do you notice about the wavelength? ","metadata":{}},{"cell_type":"code","source":"# your work for step 1 goes here","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 2","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 3","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 4","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"code","source":"# step 5","metadata":{"trusted":true},"execution_count":null,"outputs":[]},{"cell_type":"markdown","source":"## You've reached the end of the lesson. Great work!","metadata":{}},{"cell_type":"markdown","source":"## References\nhttps://www.sciencelearn.org.nz/resources/119-fundamentals-of-waves#:~:text=All%20kinds%20of%20waves%20have,%2C%20frequency%2C%20speed%20and%20amplitude.&text=A%20wave%20can%20be%20described,a%20disturbance%20that%20transfers%20energy.\n\nhttps://www.physicsclassroom.com/calcpad/waves","metadata":{}}]}
```
Comment (sebkopf) [inline week1.ipynb]: <p>double check the path, in ipython notebooks all paths are relative to the location of the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (sebkopf) [inline week1.ipynb]: <p>that looks great</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (amal6332) [inline week1.ipynb]: <p>"solve problems" is a little vague, maybe add types of problems that can be solved with the wave equation</p><p> </p><p>how will goal #2 be tested?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (amal6332) [inline week1.ipynb]: <p>Recommendation: replace "Period units are simply seconds (or minutes or hours or some time unit)." with "a period is measured in units of time (e.g., seconds, minutes, hours)</p><p> </p><p>Under: The Anatomy of a Wave, "snakey" = slinkey? or snake?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (amal6332) [inline week1.ipynb]: <p>update diagram path</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (amal6332) [inline week1.ipynb]: <p>is the dot a multiplier or dot product? If multiplier, maybe use "*"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (amal6332) [inline week1.ipynb]: <p>Another way to do this without changing the actual precision of the mathematical computation, you can use </p><pre class="ql-syntax" spellcheck="false">print("{0:.2f} mi/hr".format(v))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (amal6332) [inline week1.ipynb]: <p>Great word problem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (amal6332) [inline week1.ipynb]: <p>another way to make a dictionary (if this is helpful) is:</p><pre class="ql-syntax" spellcheck="false">ripple_vars = dict()
</pre><p>and then you can add elements to the dictionary as needed by doing</p><pre class="ql-syntax" spellcheck="false">ripple_vars["cycles] = 4
</pre><p> </p><p>you can also zip lists together into a dictionary-- for example,</p><p> </p><pre class="ql-syntax" spellcheck="false">text = ["distance btwn N &amp; A", "cycles", "time"]
vals = [1.8, 4, 8.4]
ripple_vars =  dict(zip(text, vals))

</pre><p> </p><p>Also, when doing print statements, it is generally customary to use .format() for example, </p><pre class="ql-syntax" spellcheck="false">print("The wavelength is {}".format(wl))
</pre><p>you can also use key word arguments in the placeholders,</p><pre class="ql-syntax" spellcheck="false">print("The wavelength is {wave} and the frequency is {freq}".format(w=wl, freq=f))
</pre><p>which is sometimes easier to keep track of when there is a number of placeholders</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (amal6332) [inline week1.ipynb]: <p>I really like the way your broke down the question into goal and steps here. Good job!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-aeckland2696/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (abbyeckland) [reply week1.ipynb]: <p>Lol, i guess both work...I'll go with slinky! </p>

Comment (abbyeckland) [reply week1.ipynb]: <p>good catch. supposed to be a multiplier</p>

Comment (abbyeckland) [reply week1.ipynb]: <p>I like this a lot! Thanks for the suggestion. I'm new to dictionaries but this seems cleaner than my original method.</p>

**Pull request merged**